build-system/test-configs/forbidden-terms.js

```diff
@@ -932,7 +931,6 @@ const forbiddenTermsSrcInclusive = {
       'ads/_a4a-config.js',
       'build-system/server/amp4test.js',
       'build-system/server/app-index/amphtml-helpers.js',
-      'build-system/server/app-utils.js',
```

Comment:
Was this reverted line intentional?

Reply:
Yes, #34754 removed direct mentions to `cdn.ampproject` in that file (replacing it with a regex, so now it's `cdn\\.ampproject` instead) and that also broke the checks